docs/specs/properties.md

```diff
@@ -37,6 +37,10 @@ The `y` axis represents the value interpolation factor, a value of 0
 represents the value at the current keyframe, a value of 1 represents the
 value at the next keyframe.
 
+Unlike `x` values, `y` values are not clamped to `[0 .. 1]`.  Supernormal `y`
+values allow the interpolated value to overshoot (extrapolate) beyond the
+specified keyframe values range.
+
```

Comment:
Is this correct? When I tried it a while ago it wouldn't work with `y` not in [0, 1]

Reply:
yea but does it work on lottie-web and other players?

Reply:
I works for LottieWeb and Skottie, I haven't tried other players: https://skottie.skia.org/372a0b18bde14ab9e16e0c9c8e96fa9d?e=true&h=500&l=true&w=500

I can check LottieAndroid too.

Also note that CSS' cubic Bezier has the same unclamped-Y semantics: https://cubic-bezier.com/#.93,-0.45,.47,1.22

Reply:
Also supported in LottieAndroid.